src/flp/szk.rs

```diff
@@ -221,7 +433,15 @@ where
         self.typ.joint_rand_len() > 0
     }
 
-    fn prove(
+    /// Used by a client to prove validity (according to an FLP system) of an input
+    /// that is both shared between the leader and helper
+    /// and encoded as a measurement. Has a precondition that leader_input_share
+    /// \+ helper_input_share = encoded_measurement.
```

Comment:
```suggestion
    /// + helper_input_share = encoded_measurement.
```

Reply:
This backslash escape is necessary to avoid a lint and Markdown formatting issues, see https://github.com/divviup/libprio-rs/pull/1087#discussion_r1707576356. A bare plus sign at the beginning of a line is interpreted as the start of an unordered list.